src/persist/benches/benches.rs

```diff
@@ -37,6 +38,12 @@ pub fn bench_persist(c: &mut Criterion) {
     );
     writer::bench_indexed_drain(&data, &mut c.benchmark_group("writer/indexed_drain"));
     writer::bench_compact_trace(&data, &mut c.benchmark_group("writer/compact_trace"));
+
+    let small_data = DataGenerator::small();
```

Comment:
nit: call this up at the top right under `DataGenerator::default()` so the env vars get output together instead of interspersed with the benchmark results